sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.h

```diff
@@ -98,6 +102,8 @@ typedef struct _nib_onl_entry {
      * @note    Only available if @ref GNRC_IPV6_NIB_CONF_ARSM != 0.
      */
     uint8_t l2addr[GNRC_IPV6_NIB_L2ADDR_MAX_LEN];
+    evtimer_msg_event_t reach_timeout;  /**< Event for @ref GNRC_IPV6_NIB_REACH_TIMEOUT */
+    evtimer_msg_event_t delay_timeout;  /**< Event for @ref GNRC_IPV6_NIB_DELAY_TIMEOUT */
```

Comment:
So, it's even `7*16` bytes per node after #7014 gets merged?

Here [1] is a list of all event timer types. Can I assume that every node wil eventually hold `14*16` bytes worth of event information? 

f that's true, then each node will weigh `224` bytes only for the events + the rest of the struct.
If that's still correct, then caching 5-10 neighbours will yield a net of ~1k-2k RAM.

Wouldn't you think it would be more wise to focus on another (maybe more centralized) concept, where each node only knows it's delta per event, but not the whole event timer itself?

[1] https://github.com/RIOT-OS/RIOT/blob/master/sys/include/net/gnrc/ipv6/nib.h